examples/html/landing/README.md

```diff
@@ -1,4 +1,4 @@
-Landing page of Compose for Web
+Landing page of Compose for Web (a former name of Compose HTML Library)
```

Comment:
I would use the new name as primary - `Landing page of Compose HTML (previously called Compose for Web)`
or use a note instead of brackets:
> Note that Compose HTML was previously called Compose for Web. When we refer to Compose for Web now, we mean Compose based on WebAssembly.

Reply:
Rewrited this way. is it okay?

A landing page example

> Note that this page describes Compose HTML which was previously called Compose for Web. When we refer to Compose for Web now, we mean Compose based on WebAssembly.